src/Resources/views/admin/layouts/nav-left.blade.php

```diff
@@ -0,0 +1,21 @@
+<div class="navbar-left">
```

Comment:
Due the fact that we are almost ready to release the next Bagisto version and the admin ui has changed i would like to suggest to make compatible with Bagisto master branch

Reply:
ok i 'll working on it.

Reply:
Now this is compatible with bagisto master branch. 
https://github.com/bagisto/laravel-aliexpress-dropship/pull/82/commits/91b0c60ba61e8a83ddff2b9d1526712d91b6b997